src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -903,10 +906,10 @@ void EmitAllBranches()
                     // Label to jump to when any branch completes successfully.
                     string matchLabel = ReserveName("AlternationMatch");
 
-                    // Save off runtextpos.  We'll need to reset this each time a branch fails.
-                    string startingRunTextPos = ReserveName("alternation_starting_runtextpos");
+                    // Save off pos.  We'll need to reset this each time a branch fails.
+                    string startingRunTextPos = ReserveName("alternation_starting_pos");
                     additionalDeclarations.Add($"int {startingRunTextPos} = 0;");
-                    writer.WriteLine($"{startingRunTextPos} = runtextpos;");
+                    writer.WriteLine($"{startingRunTextPos} = pos;");
```

Comment:
Could we rename "crawlpos" as well?  I don't know why we chose to name the capture stack "crawl", but "capturepos" or something like that makes more sense to me than "crawlpos".

Reply:
for sure, that will help too.